packages/shared/src/components/Feed.tsx

```diff
@@ -118,11 +119,12 @@ export default function Feed<T>({
 }: FeedProps<T>): ReactElement {
   const {
     postCardVersion,
-    postCardShareVersion,
+    postCardShareVersion: shareVersion,
     postModalByDefault,
     postEngagementNonClickable,
     showCommentPopover,
   } = useContext(FeaturesContext);
+  const postCardShareVersion = navigator.share ? ShareVersion.V2 : shareVersion;
```

Comment:
Just in case :)

```suggestion
  const postCardShareVersion = navigator?.share ? ShareVersion.V2 : shareVersion;
```

Reply:
Is there a possibility that navigator is not available? (apart from using in node.js)

Reply:
Not sure. Anyway it's not something major, we can keep it as it is :)

Reply:
Test could potentially break, we normally do check if navigator exists in other places, but not sure if it was legacy.